### PR TITLE
fork tests on other chains + fork test bug fix

### PR DESCRIPTION
**Pull request by anna-carroll**
## PR Checklist

- [ ] Added Tests
- [ ] Updated Documentation
- [ ] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/contracts-ops/contracts/test/Reboot.t.sol [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {NomadTest} from "@nomad-xyz/contracts-core/contracts/test/utils/NomadTes
 
 contract RebootTest is RebootLogic, NomadTest {
     string remote;
-    string constant _domain = "ethereum";
+    string constant testDomain = "ethereum";
 
     function setUpReboot(string memory testName) public {
         // ALL
```
[the remark this answers is not in the document]
Comment (anna-carroll) [reply packages/contracts-ops/contracts/test/Reboot.t.sol:13]: additionally, we could remove the pinned block number in order to make it more seamlessly compatible with all chains, but then the tests wouldn't cache the block which would make them even slower

**Review by anna-carroll — COMMENTED** [review]
2 annoying bits = RPC & block number

File: packages/contracts-ops/contracts/test/Reboot.t.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (prestwich) [reply packages/contracts-ops/contracts/test/Reboot.t.sol:13]: don't get RPC from config, we don't want to keep good RPCs in those

**Review by prestwich — CHANGES_REQUESTED** [review]
should be multiple PRs

- test not updater sig
- config changes
- fix writeJson in reboot setup
- refactor reboots to allow non-ethereum runs

**Comment by anna-carroll** [comment]
split up:
- https://github.com/nomad-xyz/monorepo/pull/558
- https://github.com/nomad-xyz/monorepo/pull/559
- https://github.com/nomad-xyz/monorepo/pull/560
- https://github.com/nomad-xyz/monorepo/pull/561
- https://github.com/nomad-xyz/monorepo/pull/562

**Pull request closed without merging**